### ISSUE TITLE
Comments broken 

### ISSUE THREAD

**Issue by dkhamsing** [Dimillian/SwiftHN#36]
:cry: 

![image](https://cloud.githubusercontent.com/assets/4723115/8874618/6ad31ebe-31c7-11e5-83a6-d179d75fd246.png)

**Comment by Dimillian** [comment]
:(

**Comment by Jonahss** [comment]
:(

**Comment by TekEffekt** [comment]
Broken for me too.

**Comment by Dimillian** [comment]
Fixed on the JSON_API branch, but it's a WIP

**Comment by flotteur** [comment]
Glad to see a fix in the work! Thanks Dimillian :smile:

**Comment by kishaningithub** [comment]
It's still the same.. Comments are broken

**Comment by dkhamsing** [comment]
Not sure this app is still active. I recommend Simple Reader - A free and open source client for Hacker News by Ryan Nystrom
https://itunes.apple.com/us/app/simple-reader-a-free-and-open-source-client-for-hacker-news/id1000995253?mt=8


<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Dimillian** [comment]
Yeah... I might update it if I find some time, for Swift 4 etc.... but for now it's pretty much dead.
